C:\Users\chris\AppData\Local\Programs\Python\Python36-32\python.exe C:/Git/CPS313/Week2/Exercise4.py
Enter in a number between 1 and 10: 9
IX

Process finished with exit code 0

----------------------------------------------------

C:\Users\chris\AppData\Local\Programs\Python\Python36-32\python.exe C:/Git/CPS313/Week2/Exercise4.py
Enter in a number between 1 and 10: 11
Error: Input outside range.

Process finished with exit code 0